docs/mkdocs.yml

```diff
@@ -63,6 +63,7 @@ nav:
   - how-to/per-hostedcluster-dashboard.md
   - how-to/metrics-sets.md
   - how-to/troubleshooting-general.md
+  - how-to/inspect-payload.md
```

Comment:
does this belong to troubleshooting? or even https://hypershift-docs.netlify.app/how-to/aws/troubleshooting/debug-nodes/

Reply:
No strong feelings on this, idm either or.

Reply:
Maybe https://github.com/openshift/hypershift/blob/main/docs/content/how-to/troubleshooting-general.md since this isn't cloud specific?